spyder/plugins/console/tests/test_plugin.py

```diff
@@ -80,5 +77,67 @@ def test_completions(console_plugin, qtbot):
     assert u'import' in shell.toPlainText()
 
 
+def test_handle_exception(console_plugin, mocker):
+    """Test that error dialog is called """
```

Comment:
`"""Test that error dialog is called """`->

`"""Test that error dialog is called."""`

Reply:
Fixed